NameLayerMain/src/vg/civcraft/mc/namelayer/GroupManager.java

```diff
@@ -58,9 +62,17 @@ public int createGroup(Group group){
 			NameLayerPlugin.log(Level.INFO, "Group create was cancelled for group: " + group.getName());
 			return -1;
 		}
-		int id = groupManagerDao.createGroup(
+		int id;
+		if (savetodb){
+			id = groupManagerDao.createGroup(
 				event.getGroupName(), event.getOwner(), 
 				event.getPassword());
+			Mercury.message("create " + group.getName() +" "+ 
```

Comment:
This Mercury call also needs to be wrapped.

As I think of it, actually, you'll probably need to wrap all mentions of Mercury w/ a NameLayer-local accessory class, or all of this code will fail at runtime for non-sharded servers.